login.html

```diff
@@ -16,17 +15,19 @@
 	<body>
 		<main class="userForm_main">
 			<a href="/">
-				<img src="./image/logo.png" alt="판다마켓 로고" id="user_form_logo" />
+				<img src="./image/logo.png" alt="판다마켓 로고" class="user_form_logo" />
 			</a>
 			<form action="#" class="user_form">
 				<label for="email">이메일</label>
-				<input type="email" id="email" name="email" placeholder="codeit@email.com" />
+				<input type="email" id="email" name="email" placeholder="이메일을 입력해주세요" />
 				<label for="password">비밀번호</label>
 				<div class="input_box">
-					<input type="password" id="password" name="password" />
-					<img src="./image/btn_visibility_on.png" alt="비밀번호 보기" class="visible_on" />
+					<input type="password" id="password" name="password" placeholder="비밀번호를 입력해주세요" />
+					<button class="visible_button">
```

Comment:
❗️ 수정요청
버튼 태그는 type 속성을 명시해주지 않으면 submit 을 기본으로 가지게됩니다.
submit 타입의 버튼은 form 태그를 제출하는 역할을 하기때문에 해당 태그에는 button 타입이 적절합니다~

```suggestion
					<button type="button" class="visible_button">
```

type 속성은 버튼 동작에 있어 중요한 속성이므로 습관적으로 기입해주시는 것을 추천드려요~